hack/create-installation.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+SOURCE_PATH="$(dirname $0)/.."
+TMP_DIR="$(mktemp -d)"
+INSTALLATION_PATH="${TMP_DIR}/installation.yaml"
+
+REGISTRY="$(${SOURCE_PATH}/hack/get-cd-registry.sh)"
+COMPONENT_NAME="$(${SOURCE_PATH}/hack/get-cd-component-name.sh)"
+
+cat << EOF > ${INSTALLATION_PATH}
+apiVersion: landscaper.gardener.cloud/v1alpha1
+kind: Installation
+metadata:
+  name: os-gardenlinux
+spec:
+  componentDescriptor:
+    ref:
+      repositoryContext:
+        type: ociRegistry
+        baseUrl: ${REGISTRY}
+      componentName: ${COMPONENT_NAME}
+      version: ${EFFECTIVE_VERSION}
+
+  blueprint:
+    ref:
+      resourceName: os-gardenlinux-controller-registration
```

Comment:
Since some time a higher level resource ControllerDeployment is used https://github.com/gardener/gardener-extension-os-gardenlinux/blob/0b5c895c3b3596ec3f656c72f156c9983c9c1503/example/controller-registration.yaml#L3

```suggestion
      resourceName: os-gardenlinux-controller-deployment
```

Reply:
We have changed this. Also changed the corresponding name of the blueprint resource in resources.yaml-